psscriptanalyzer/Manage-Users.ps1

```
@@ -0,0 +1,38 @@
+    Write-Host "creating user $Username"
+    $secure = ConvertTo-SecureString -String $Password -AsPlainText -Force
+    $cred = New-Object System.Management.Automation.PSCredential($Username, $secure)
+    New-LocalUser -Name $Username -Password $secure -PasswordNeverExpires
+    Add-LocalGroupMember -Group "Administrators" -Member $Username
+    return $cred
+}
+
+function Connect-AdminSession {
+    param(
+        [string]$ComputerName,
+        [string]$AdminPassword = "AdminP@ss2024"
```

PSScriptAnalyzer (1.25.0)

[warning] Parameter '$AdminPassword' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $AdminPassword type to SecureString; Set $AdminPassword type to PSCredential

(PSAvoidUsingPlainTextForPassword)